happypose/pose_estimators/megapose/inference/icp_refiner.py

```diff
@@ -139,6 +139,12 @@ def icp_refinement(
     cam_K,
     TCO_pred,
     n_min_points=1000,
+    min_measured_depth=0.2,
+    max_measured_depth=5.0,
+    iterations=100,
+    tolerance=0.05,
+    rejection_scale=2.5,
+    num_levels=4,
```

Comment:
Can you document them?
It is not immediately clear to me what they all corresponds to.

Reply:
I see now that there are some description of those parameters in https://github.com/agimus-project/happypose_ros/pull/23

Reply:
As discussed. Let's keep it as is since the external API is documented on the ROS side